src/application.ts

```diff
@@ -83,7 +86,7 @@ export default class Application implements Owner {
     registry.register(`environment:/${this.rootName}/main/main`, Environment);
     registry.registerOption('helper', 'instantiate', false);
     registry.registerOption('template', 'instantiate', false);
-    registry.register(`document:/${this.rootName}/main/main`, window.document as any);
+    registry.register(`document:/${this.rootName}/main/main`, this.document as any);
```

Comment:
Yes, because ts will complain: ` Argument of type 'Document' is not assignable to parameter of type 'FactoryDefinition<any>'.`